gameMechanics.js

```diff
@@ -0,0 +1,303 @@
+// Game Mechanics for the Connect Four Game
+
+// Main methods
+
+var createGrid = function(height, width) {
+	var grid = new Array(width);
```

Comment:
In JAVA, you have to declare `new Array()`.  In Javascript you need only say `[]`.